doc/source/notebooks/intro.ipynb

```diff
@@ -57,22 +57,22 @@
     "\n",
     "In this section, we show how GPflow's utilities and codebase can be used to build new probabilistic models.\n",
     "These can be seen as complete examples.\n",
-    "  - [Kernel design](tailor/kernel_design.ipynb) Shows how to implement a covariance function that is not available by default in GPflow. For this example, we looked at the Brownian motion covariance.\n",
+    "  - [Kernel design](tailor/kernel_design.ipynb) shows how to implement a covariance function that is not available by default in GPflow. For this example, we look at the Brownian motion covariance.\n",
     "<!--  - [likelihood design](tailor/likelihood_design.ipynb) **[TODO]** -->\n",
     "<!--  - [Latent variable models](tailor/models_with_latent_variables.ipynb) **[TODO]** -->\n",
     " <!-- - [Updating models with new data](tailor/updating_models_with_new_data.ipynb) **[TODO]** -->\n",
     "  - [External mean functions](tailor/external-mean-function.ipynb): here is how to use a neural network as mean function.\n",
-    "  - [Mixture density network](tailor/mixture_density_network.ipynb): we show how we can use GPflow's utilities to build a non GP probabilistic model in no time.\n",
+    "  - [Mixture density network](tailor/mixture_density_network.ipynb): we show how we can use GPflow's utilities to build a non-GP probabilistic model in no time.\n",
```

Comment:
>  to build a non-GP probabilistic model in no time

Slightly subjective :stuck_out_tongue:

Reply:
Well, tell that to the person who wrote it :)

Reply:
```suggestion
    "  - [Mixture density network](tailor/mixture_density_network.ipynb): we show how we can use GPflow's utilities to build a non-GP probabilistic model.\n",
```

Reply:
How about
```suggestion
    "  - [Mixture density network](tailor/mixture_density_network.ipynb): we show how GPflow's utilities make it easy to build other, non-GP probabilistic models.\n",
```
instead?

Reply:
I like it :grin: